README.md

```diff
@@ -87,29 +87,44 @@ To see a sample of what FlamingPy can do, let us first import a few important ob
 
 ```
 from flamingpy.codes import SurfaceCode
-from flamingpy.noise import CVLayer
+from flamingpy.cv.ops import CVLayer
 from flamingpy.decoders import correct
 ```
 
-Next, let us instantiate an RHG lattice -- the measurement-based version of the surface code:
+Next, let us instantiate an RHG lattice – the measurement-based version of the surface code:
 
 ```
 RHG = SurfaceCode(3)
 ```
 
-The integer denotes the code distance. By default, the boundaries are set to "open". Now, let us define and apply a continuous-variable noise model to the code:
+The integer denotes the code distance. By default, the boundaries are set to “open”. Next, let us associate the nodes in the RHG lattice with CV states:
```

Comment:
Lets stick to more regular characters, sometimes there are issues when rendering fancy characters like the quotes you added. This might not be the case but just for sanity, even if someone decides to copy–paste to python code.
```suggestion
The integer denotes the code distance. By default, the boundaries are set to "open". Next, let us associate the nodes in the RHG lattice with CV states:
```